get-started/setup-an-ide.md

```diff
@@ -1,7 +1,95 @@
 # Setup an IDE
 
-* vs code
-* goland
-* vim \(optional\)
-* emacs \(optional\)
+## vs code
 
+## GoLand
+
+You can use [GoLand](https://www.jetbrains.com/go/) to easily run or debug TiDB in many situations.
+
+> Goland uses [delve](https://github.com/go-delve/delve) to debug go programs, usually GoLand has an usable delve and needn't any additional configuration. But in some cases, you may need to download a delve by yourself.
+
+Step 1. Setup `go` environment, see previous sections for more information.
+
+Step 2. Download `GoLand` [here](https://www.jetbrains.com/go/download/) and install it.
+
+Step 3. Open the TiDB project in GoLand.
+
+Step 4. Add these config files to `<tidb-dir>/.idea/runConfigurations/` directory:
+
+- unistore_4000.xml
+```xml
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="unistore 4000" type="GoApplicationRunConfiguration" factoryName="Go Application">
+    <module name="tidb" />
+    <working_directory value="$PROJECT_DIR$" />
```

Comment:
does the user need to replace `$PROJECT_DIR$` with a concert directory path? please directly point out this information